buildhat/motors.py

```diff
@@ -520,3 +530,60 @@ def run_to_position(self, degreesl, degreesr, speed=None, direction="shortest"):
         th2.start()
         th1.join()
         th2.join()
+
+
+class Motors:
+    def __init__(self, *ports):
+        """Initialise motor"""
+        Device._setup()
+        self._ports = []
+        for port in ports:
+            self._ports += [Motor(port)]
+        self.default_speed = 20
+
+    def run_for_degrees(self, degrees):
+        speed = self.default_speed
+        cmd = ""
+        for motor in self._ports:
+            cmd += motor._run_for_degrees(degrees, speed, run=False)
+        self._write(f"{cmd}\r")
+
+    def run_for_seconds(self, seconds):
+        speed = self.default_speed
+        cmd = ""
+        for motor in self._ports:
+            cmd += motor._run_for_seconds(seconds, speed, run=False)
+        self._write(f"{cmd}\r")
+
+    def run_to_position_ok(self, degrees, direction="shortest"):
+        speed = self.default_speed
+        cmd = ""
+        for motor in self._ports:
+            cmd = motor._run_to_position(degrees, speed, direction, run=False)
+            self._write(f"{cmd}\r")
+
+    def run_to_position(self, degrees, direction="shortest"):
+        speed = self.default_speed
+        cmd = ""
+        for motor in self._ports:
+            cmd += motor._run_to_position(degrees, speed, direction, run=False)
+        self._write(f"{cmd}\r")
+
+    def start(self):
+        speed = self.default_speed
+        cmd = ""
+        for motor in self._ports:
+            setup = f"port {motor.port} ; combi 0 1 0 2 0 3 0 ; select 0 ; pid {motor.port} 0 0 s1 1 0 0.003 0.01 0 100;\r"
+            self._write(setup)
+            cmd += f"port {motor.port} ; set {speed} ; "
```

Comment:
We had a problem in PR #125 where combi initialization was causing a noticeable lag in motor pairings, so it was separated out from the set speed.  Perhaps loop the motor list again and put the set command in there so there's not as much lag between the starting motor movements?